src/trace_read.h

```diff
@@ -53,7 +53,7 @@ POSSIBILITY OF SUCH DAMAGE.
 ///////////////////////////////////////////////////////////////////////////////////////////////
 
 
-#define MAX_TR_REG 300 // REG_LAST = 298
+#define MAX_TR_REG 321 // REG_LAST = 298 // need to check!!! 
```

Comment:
need to check? delete the comment after confirmation

Reply:
@hyesoon Please review this

Reply:
@samjijina can you count and double check?

Reply:
done resolved